enkit/outputs/commands.go

```diff
@@ -303,5 +303,24 @@ func NewShutdown(root *Root) *Shutdown {
 }
 
 func (c *Shutdown) Run(cmd *cobra.Command, args []string) error {
-	return fmt.Errorf("`enkit outputs Shutdown` is unimplemented")
+	bbOpts := bbexec.NewClientOptions(
+		c.root.Log,
+		0, /* tunnel port does not matter in this case */
+		c.root.OutputsRoot,
+	)
+	bbClient, err := bbexec.MaybeStartClient(bbOpts)
+	if err != nil {
+		return err
+	}
+	var errs []error
+	if err := bbClient.Shutdown(); err != nil {
+		errs = append(errs, fmt.Errorf("error maybe? killing the process of existing bb_clientd %v", err))
+	}
+	if err := os.RemoveAll(c.root.OutputsRoot); err != nil {
+		errs = append(errs, err)
+		c.root.Log.Errorf("error removing output root %s %v", c.root.OutputsRoot, err)
```

Comment:
This should be the only real error - if we fail to clean the FS as expected the command should fail. The other steps (fusermount, killing bb_clientd) are supposed to increase the likelihood that the FS clean goes well